spec/unit/puppet/type/iis_site_spec.rb

```diff
@@ -38,26 +38,6 @@
     end
   end
 
-  # TODO: fix tests below
-  context "parameter :physicalpath" do
```

Comment:
I'll restore that. Mislead by: 
https://technet.microsoft.com/en-us/library/hh867858(v=wps.630).aspx